### DISCUSSION TITLE
Hydrogen portable in a housing enclosure????

### DISCUSSION THREAD

**Discussion by Minklocsp12builder** [hydrogen-music/hydrogen#1638] [Ideas]
I'm interested in using Hydrogen as a portable drum machine sampler like a EMU SP1200 with a small LCD DISPLAY screen is it possible?
minkottophillips@gmail.com

**Comment by theGreatWhiteShark** [comment]
Hey @Minklocsp12builder ,

Sure. You can build and run it on a Raspberry Pi and attach a small LCD Display.

**Reply by Minklocsp12builder** [reply]
Thanks I have a sp1200 faceplate I just want to bring it to life 
basically a portable drum machine sampler.
